.github/workflows/dispatch-build-bottle.yml

```diff
@@ -276,6 +276,7 @@ jobs:
         env:
           GH_TOKEN: ${{secrets.HOMEBREW_GITHUB_PUBLIC_REPO_TOKEN}}
         run: |
+          # shellcheck disable=SC2016
```

Comment:
@carlocab Could this get a comment explaining why it's being disabled? Thanks ❤️

Reply:
#127989